### DISCUSSION TITLE
cannot import name 'relax' from 'tvm' (/home/xcn/new-data/lzx/tvm/python/tvm/__init__.py)

### DISCUSSION THREAD

**Discussion by lizexu123** [mlc-ai/mlc-zh#78] [General]
这个TVM官网还没有，怎么导入呢

**Comment by Hzfengsy** [comment]
可以通过`python3 -m pip install mlc-ai-nightly -f https://mlc.ai/wheels`进行安装

**Reply by lizexu123** [reply]
我在Ubuntu上运行过，确实还是不行

**Reply by Hzfengsy** [reply]
如果相同的环境装有其他版本TVM且设置PYTHONPATH的情况下可能会导致import优先级问题，可以尝试重置PYTHONPATH

**Reply by lizexu123** [reply]
我感觉是不是TVM0.9.0没有这个relax，是relay，所以导入不成功呢

**Reply by Hzfengsy** [reply]
`mlc-ai-nightly`并不是依据TVM 0.9编译的，如果本地装有0.9可以考虑卸载，或者重置PYTHONPATH

**Reply by lizexu123** [reply]
哦，明白了，确实装有TVM0.9.0，我试试，感谢

**Reply by lizexu123** [reply]
确实成功了，原来不需要安装TVM，可能我的环境变量里面设置了TVM的python变量，十分感谢
